### ISSUE TITLE
Umami on Netlify stop working since v1.37.0 upgrade

### ISSUE THREAD

**Issue by laurent-d** [umami-software/umami#1444]
Hello, 
My Umami stopped to worked and collect data since the 1.37.0 upgrade (it's works smoothly from March 2022).

On the Dashboard the only things i have an error is the realtime view on console log : 
```
react_devtools_backend.js:4026 TypeError: Failed to fetch
    at r (_next/static/chunks/pages/_app-3f39117fa6aacd01.js:1:434499)
    at o (_next/static/chunks/pages/_app-3f39117fa6aacd01.js:1:434853)
    at _next/static/chunks/398-8eb46fcd2cf61446.js:1:19669
    at s (_next/static/chunks/main-f1f37299d5d02885.js:1:104683)
    at Generator._invoke (_next/static/chunks/main-f1f37299d5d02885.js:1:104471)
    at forEach.e.<computed> [as next] (_next/static/chunks/main-f1f37299d5d02885.js:1:105106)
    at a (_next/static/chunks/pages/_app-3f39117fa6aacd01.js:1:454515)
    at u (_next/static/chunks/pages/_app-3f39117fa6aacd01.js:1:454718)
    at _next/static/chunks/pages/_app-3f39117fa6aacd01.js:1:454777
    at new Promise (<anonymous>)
```
    
issue on fetch .. _app-3f39117fa6aacd01.js:1          GET https://hello-laurent.netlify.app/api/realtime/init 500

I have no issue during the deploy on Netlify.
My Post-gre DB run on Supabase

If i made  on local
yarn prisma migrate resolve --rolled-back "02_add_event_data"
yarn prisma migrate deploy
yarn check-db

I ve got this issue 

```
2 migrations found in prisma/migrations

Applying migration `02_add_event_data`
Error: P3018

A migration failed to apply. New migrations cannot be applied before the error is recovered from. Read more about how to resolve migration issues in a production database: https://pris.ly/d/migrate-resolve

Migration name: 02_add_event_data

Database error code: 42501

Database error:
ERROR: must be owner of table event

DbError { severity: "ERROR", parsed_severity: Some(Error), code: SqlState(E42501), message: "must be owner of table event", detail: None, hint: None, position: None, where_: None, schema: None, table: None, column: None, datatype: None, constraint: None, file: Some("aclchk.c"), line: Some(3583), routine: Some("aclcheck_error") }
```

Someone have an idea about it or possibility to run Migration manually from Supabase manager ?

**Comment by AkashRajpurohit** [comment]
I had the same issues with Supabase when tried running migrations locally and while trying to fix it, ended up breaking lots of stuff so had to create a new db, migrate all the data to the new db, but I'm still stuck with [the issues and endpoints throwing 5xx errors](https://github.com/umami-software/umami/discussions/1434#discussioncomment-3439643) 😞

**Comment by AkashRajpurohit** [comment]
Update: https://github.com/umami-software/umami/discussions/1434#discussioncomment-3439712
Post this, I am able to fix the issues and getting no errors when running check-db and update-db commands locally.

**Comment by laurent-d** [comment]
Thanks for the help @AkashRajpurohit !!!
i will look in quickly.

**Comment by azat-io** [comment]
Same error after clean install

**Comment by laurent-d** [comment]
Sorry @mikecao to mention you.
But do you think the next release could fix 1.37 issue upgrade ?

**Comment by laurent-d** [comment]
Thanks a lot @mikecao for the hard work !!!
Have a good day from France
